specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-05-01/machineLearningServices.json

```diff
@@ -5519,6 +5561,7 @@
         "nodeSelector": {
           "type": "object",
           "description": "Node Selector",
+          "x-nullable": true,
           "additionalProperties": {
             "type": "string"
```

Comment:
are the values of this dictionary nullable? ie is something like `{"key1": null}` possible? if so, mark the additionalProperties as x-nullable, too:
```json
"additionalProperties": {
  "type": "string",
  "x-nullable": true
}
```

Reply:
Yes, fixed